src/filter/s0rter/s0rter.c

```diff
@@ -0,0 +1,255 @@
+#include "s0rter.h"
```

Comment:
You should include a comment header with the copyright and license (preferably some version of GPL).